assignments/sidharth/week1/1.3.txt

```diff
@@ -0,0 +1 @@
+(define (sum-of-squares-bigger a b c) (cond ((and (> a c) (> b c)) (+ (* a a) (* b b))) ((and (> b a) (> c a)) (+ (* b b) (* c c))) ((and (> c b) (> a b)) (+ (* a a) (* c c))))) 
```

Comment:
This doesn't work when the operands are equals, e.g. (sum-of-squares-bigger 2 2 2). I think you need to replace '>' by '>='.